arches_search/indexing/indexers/number.py

```diff
@@ -15,13 +15,14 @@ def index(self, tile, node):
         nodeid = str(node.nodeid)
         document = {"numbers": [], "strings": []}
         self.datatype.append_to_document(document, tile.data[nodeid], node, tile)
+        search_items = []
         for number_object in document["numbers"]:
-            numeric_search = NumericSearch.objects.create(
+            numeric_search = NumericSearch(
                 node_alias=node.alias,
                 tileid_id=tile.tileid,
                 resourceinstanceid_id=tile.resourceinstance_id,
                 datatype=self.datatype.datatype_name,
                 graph_slug=node.graph.slug,
                 value=number_object["number"],
             )
-            numeric_search.save()
+            search_items.append(numeric_search)
```

Comment:
Think you need to return search_items in this indexer.  Looks like you just missed this one.